.goreleaser.yaml

```diff
@@ -187,6 +207,15 @@ changelog:
     exclude:
       - "^docs:"
       - "^test:"
+dockerhub:
+  - images:
+      - Dynatrace/dynatrace-otel-collector
+    username: "tbd"
+    secret_name: "tbd"
```

Comment:
I would opt for the default secret names, then they don't need to be defined here.
Defaults are `DOCKER_USERNAME` and `DOCKER_PASSWORD`
[source](https://goreleaser.com/customization/dockerhub/)